lib/activerecord-multi-tenant/model_extensions.rb

```diff
@@ -38,11 +38,11 @@ def primary_key
 
           def inherited(subclass)
             super
-            MultiTenant.register_multi_tenant_model(subclass.table_name, subclass) if subclass.table_name
+            MultiTenant.register_multi_tenant_model(subclass)
```

Comment:
This seems backwards incompatible. Shouldn't the register push `[table_name, subclass]` to `@@multi_tenant_models`? Why was table_name previously a parameter?

Reply:
@serprex  `@@multi_tenant_models` is only used by `.multi_tenant_model_for_table` to resolve the table name and get the model. Since it is an internal variable, I don't see any problem with the structure changing.
`.register_multi_tenant_model` is a method we use internally, but since it is a public method, I reverted the argument back for backward compatibility.

> Why was table_name previously a parameter?

Introduced to resolve model from table_name. 3832bb87a3e6ec24c
Compared to this change, the only difference is whether it is static or dynamic.

Reply:
Strict compatibility no longer important as this will be in 2.0